README.md

```diff
@@ -70,6 +70,7 @@ reminder.remind();
 
 ## Change Log
 
-### 1.2.0
+### 1.2.2
```

Comment:
Please if you could add a separate heading for the new version 🙂  This way it's easier to see what happened in each version

Reply:
I will change it. thanks !